### PR TITLE
Removing submodule in favour of rust crate

### PR DESCRIPTION
**Pull request by tomusdrw**
Fixes issues with `git submodule update --init` needed after `git pull`
Cargo will now rebuild the crate if there were some changes in UI code.

### CODE REVIEW

File: ethcore/res/ethereum/tests [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 97066e40ccd061f727deb5cd860e4d9135aa2551
+Subproject commit 862b4e3d4a9a7141af1b4aaf7dfe228a6a294614
```
Comment (jacogr) [inline ethcore/res/ethereum/tests]: Does this need to be here?

Comment (tomusdrw) [inline ethcore/res/ethereum/tests]: Nope, Submodules :/

**Comment by jacogr** [comment]
Looks like it does exactly what we discussed. Will need to get it in and see how it goes.

**Comment by jacogr** [comment]
Merged in master.

**Pull request merged**